docs/deployment.md

```diff
@@ -35,49 +36,62 @@ RelayServer deployments consist of multiple instances (pods) of different contai
-  It increases and decreases based on how many clients send how many requests against the relayed targets.
-  
-  **DO** run at least 2 instances of this container on different machines in order to provide a reliable system.  
+  provides the endpoints for any client to send a request to a target. The load of this component is highly dynamic. It
+  increases and decreases based on how many clients send how many requests against the relayed targets.
+
+  **DO** create your own project using the RelayServer v3 NuGet packages. Use the corresponding persistence packages for
+  the database of your choice.  
+  **DO** run at least 2 instances of this service on different machines in order to provide a reliable system.  
   Consider providing automatic scaling functionality of this component based on load.
 
   Configuration:
   * Provide a shared volume to all instances of Thinktecture.Relay.IdentityServer i.e. on /var/bodystore and add a
-    configuration property for the storage path to this directory
-  * Provide a connection string to the RelayServer database 
+  configuration property for the storage path to this directory
+  * Provide a url to the RelayServer.IdentityServer component
+  * Provide a connection string to the RelayServer database
   * Provide a connection string to the RabbitMQ
 
 * Thinktecture.Relay.ManagementApi  
-  This component is used to manage entries in the RelayServer database, i.e. add a configuration for a new connector.
-  These actions usually are not performed too often and load is expected to be idle for most of the time.  
+  This component is used to manage entries in the RelayServer database, i.e. add a configuration for a new tenant / 
+  connector. These actions usually are not performed too often and load is expected to be idle for most of the time.  
   If not absoluteley required we suggested not exposing this container to the public internet. Have it available only
   for your internal administrative usages. Unless you have higher load and reliability requirements and/or perform a
   lot of automatic provisioning of connectors, it should be fine to run only a single instance.
-  
+
+  **DO** create your own project using the RelayServer v3 NuGet packages. Use the corresponding persistence packages for
+  the database of your choice.  
+  **Do NOT** use the example / development project and docker file from this repository in production.  
+
   Configuration:
   * Provide a connection string to the RelayServer database 
 
 * Thinktecture.Relay.StatisticsApi  
   This component provides endpoints to retrieve usage data about the system from the database. For example you could
   fetch this data in intervals and feed it into your monitoring or billing systems. Load on this component is based
-  on your internal usage only and should be fairly constant.
+  on your internal usage only and should be fairly constant.  
```

Comment:
Please remove the trailing white spaces.

Reply:
Same thing, this is needed to enforce a line-break in the rendering.